src/index.html

```diff
@@ -3,11 +3,236 @@
+                src="/images/details/2.png"
+                alt="Bicycle with automatic unlock image - 2"
+              >
+            </div>
+            <h4 class="section__element-title section__element-title--details">Auto unlock</h4>
+            <p class="section__description section__description--details">
+              The app senses when you're nearby to unlock automatically. GPS tracking so you know where your bike is and can track it anytime.
+            </p>
+          </div>
+
+          <div class="section__details">
+            <div class="section__details-img section__details-img--3 section__details-img--long">
+              <img
+                class="visually-hidden"
+                src="/images/details/3.png"
+                alt="Bike with integrated range lights"
+              >
+            </div>
+            <div class="section__details-img section__details-img--4">
+              <img
+                class="visually-hidden"
+                src="/images/details/4.png"
+                alt="Bicycle that has integrated range lights - image 2"
+              >
+            </div>
+            <h4 class="section__element-title section__element-title--details">Range & Integrated lights</h4>
+            <p class="section__description section__description--details">
+              The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+            </p>
+          </div>
+
+          <div class="section__details">
+            <div class="section__details-img section__details-img--5 section__details-img--long section__details-img--special">
+              <img
+                class="visually-hidden"
+                src="/images/details/5.png"
+                alt="Bike with hydraulic brakes"
+              >
+            </div>
+            <div class="section__details-img section__details-img--special-long section__details-img--6">
+              <img
+                class="visually-hidden"
+                src="/images/details/6.png"
+                alt="Bicycle that has hydraulic disk brakes image - 2"
+              >
+            </div>
+            <h4 class="section__element-title section__element-title--details">Hydraulic disc brakes & Lightweight</h4>
+            <p class="section__description section__description--details">
+              Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+            </p>
+          </div>
+
+          <a href="#" class="button section__button button--324">Explore</a>
+        </section>
+
+        <section class="section main__section main__section--m-biggest">
+          <h2 class="section__title" id="contact">Contact us</h2>
+
+          <div class="section__details">
+            <div class="form">
```

Comment:
My form has validation, but I disabled the submission with JS to prevent reloading. `Page shouldn't be reloaded on form submit (https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault)` Now I disabled the rule and you can see the validation.